test/playwright/test_ux.py

```diff
@@ -89,40 +88,77 @@ def _create_new_environment(page, screenshot=False):
     # ensure new filename in case this test is run multiple times
     new_env_name = f'test_env_{random.randint(0, 100000)}' 
     # set timeout for building the environment
-    time_to_build_env = 2 * 60 * 1000  # 2 minutes in milliseconds
+    time_to_build_env = 3 * 60 * 1000  # 3 minutes in milliseconds
 
     # Create the new environment
     # click the + to create a new env
     page.get_by_label("Create a new environment in the username namespace").click()
     if screenshot:
-        page.screenshot(path="test-results/conda-store-new-env.png")
+        page.screenshot(path="test-results/create-new-env.png", clip={'x': 0, 'y': 145, 'width': 275, 'height': 50})
     # fill in the env name
     page.get_by_placeholder("Environment name").fill(new_env_name)
     # fill in the description
     page.get_by_placeholder("Enter here the description of your environment").fill("description")
+    if screenshot:
+        page.screenshot(path="test-results/add-package-button.png", clip={'x': 300, 'y': 385, 'width': 425, 'height': 170})
+        page.screenshot(path="test-results/name-description.png")
     # click the + to add a package
     page.get_by_role("button", name="+ Add Package").click()
     # add a package to the ui
     page.get_by_label("Enter package").fill("rich")
+    if screenshot:
+        page.screenshot(path="test-results/package-selection.png")
     page.get_by_role("option", name="rich", exact=True).click()
+
+    # add version spec
+    page.get_by_role("row", name="rich").get_by_role("button").first.click()
+    
+    if screenshot:
+        page.screenshot(path="test-results/package-version-constraint.png")
+    page.get_by_role("option", name=">", exact=True).click()
+    # page.get_by_role("cell", name="> ​").get_by_role("button").nth(1).click()
+    page.get_by_role("cell", name=">").get_by_role("button").nth(1).click()
+    if screenshot:
+        page.screenshot(path="test-results/package-version-number.png")
+    page.get_by_role("option", name="12.5.1", exact=False).click()
+
     # open up the channels accordian card
     page.get_by_role("button", name="Channels").click()
     # click the + to add a channel
     page.get_by_role("button", name="+ Add Channel").click()
     # fill in conda-forge as the new channel name
     page.get_by_label("Enter channel").fill("conda-forge")
+    if screenshot:
+        page.screenshot(path="test-results/add-channel.png", clip={'x': 300, 'y': 440, 'width': 425, 'height': 202})
     # press enter to submit the channel to the list
     page.get_by_label("Enter channel").press("Enter")
+    # switch to yaml editor
+    page.get_by_label("YAML").click()
+    if screenshot:
+        page.screenshot(path="test-results/yaml-editor.png")
+    # switch back
+    page.get_by_label("GUI", exact=False).click()
+    
+    if screenshot:
+        page.screenshot(path="test-results/create-button.png")
```

Comment:
We may need to pagedown (and clip?) this image to highlight the "Create" button. It's currently not seen in the capture.

Ref: https://github.com/conda-incubator/conda-store/pull/731/files#diff-d3e83a1ffffabcea8024c779ef4294cd88f25568325e6e6baf7ff719030c3c0b